core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/recoverer.go

```diff
@@ -595,7 +595,7 @@ func (r *logRecoverer) tryExpire(ctx context.Context, ids ...string) error {
 	if err != nil {
 		return fmt.Errorf("failed to get states: %w", err)
 	}
-	lggr := r.lggr.With("where", "clean")
+	lggr := r.lggr.Named("TryExpire")
```

Comment:
🏄 I would feel inclined to derive a new logger at the very beginning of a function, even if it is a wasted operation in the error scenarios above